dojo/tools/awssecurityhub/compliance.py

```diff
@@ -14,6 +14,7 @@ def get_item(self, finding: dict, test):
         epss_score = None
         mitigation = finding.get("Remediation", {}).get("Recommendation", {}).get("Text", "")
         description = "This is a Security Hub Finding \n" + finding.get("Description", "")
+        description += f"\n**AWS Finding ARN:** {finding_id}\n"
```

Comment:
Could you please also add values like "Region" to the findings?

Reply:
Hi @manuel-sommer I checked inspector.py and guardduty.py to make sure Description would capture ARN for each.  I also tried to extend compliance.py to capture Region + AwsAccountId as suggested.  I modeled it after guardduty.py description so now all 3 (guardduty, compliance, inspector) should all contain ARN + AwsAccountId + Region in the Description fields.  

I appreciate the suggestions and guidance!